ext/net/ops.rs

```diff
@@ -608,6 +611,145 @@ where
   net_listen_udp::<NP>(state, addr, reuse_address, loopback)
 }
 
+#[cfg(unix)]
+#[op2(async, stack_trace)]
+#[serde]
+pub async fn op_net_connect_vsock<NP>(
+  state: Rc<RefCell<OpState>>,
+  #[smi] cid: u32,
+  #[smi] port: u32,
+) -> Result<(ResourceId, (u32, u32), (u32, u32)), NetError>
```

Comment:
Flatten the return tuple?

Reply:
all the accept methods are (rid, addr, addr, fd) for handling in the js, so they need to be grouped.